source/s3_client.c

```diff
@@ -151,32 +152,9 @@ uint32_t aws_s3_client_get_max_active_connections(
     struct aws_s3_client *client,
     struct aws_s3_meta_request *meta_request) {
     AWS_PRECONDITION(client);
+    (void)meta_request;
```

Comment:
why not just remove `meta_request` from the args, it's a private helper.

Or do we even need this helper? Can we just have it as a member of client? It won't change after the client initialized now, right?

Reply:
I'm pretty sure we'll be using meta_request again in the near future. If we're going to use different magic numbers for S3-Express vs S3-Vanilla, there will be some `meta-request->storage_class` variable that we'll need to check